### ISSUE TITLE
Unreachable to load default config if LGK is not found.

### ISSUE THREAD

**Issue by blkcatman** [Looking-Glass/HoloPlayJS_Issues#8]
Unreachable to load default config if LGK is not found in PC.

fixed codes in holoplay.js from [release 0.2.1](https://lookingglassfactory.com/downloads/three-js-library-looking-glass/
)
```holoplay.js
--- a/holoplay.js
+++ b/holoplay.js
@@ -212,7 +212,7 @@ function HoloPlay(scene, camera, renderer, focalPointVector, constantCenter, hiR
     
        function applyCalibration (calibration_obj)
        {
-        if(calibration_obj === undefined || calibration_obj === ""){
+        if(!calibration_obj){
             jsonObj = defaultCalibration;
             alert("No Looking Glass display connected; using default calibration data. Please ensure your Looking Glass is connected to your computer via USB and reload the page.")
         } else {
@@ -247,6 +247,8 @@ function HoloPlay(scene, camera, renderer, focalPointVector, constantCenter, hiR
                        } else { 
                                alert(errstr);
                        }
+                       applyCalibration(null);
+                       initialized = true;
                        finished();
                }, 800);
                ws.onmessage = function(event) {
```

**Comment by alxdncn** [comment]
Thanks for posting this! I don't quite understand where the issue is and I can't replicate it. What happens when you don't have LKG accessible?

**Comment by blkcatman** [comment]
> What happens when you don't have LKG accessible?

I'm sorry, my comment is wrong.
holoplay.js don't load a default calibration config if it cannot find a local WebSocket server.
If it do not want to load the default config when the server is not found, it's correct.
But if you don't assume that, it needs to be fixed as above.

**Comment by alxdncn** [comment]
I see, that makes sense. I'll look into this and try to patch it for 0.2.2!

**Comment by alxdncn** [comment]
Fixed on 0.2.2, now live
